cmd/oj/main.go

```diff
@@ -5,6 +5,7 @@ package main
 import (
 	"flag"
 	"fmt"
+	"io"
 	"io/ioutil"
```

Comment:
Not related to this PR, but ioutil is deprecated, you can refactor

Reply:
Time change. Updated to use `os.ReadFile` instead.